test/test/alloc/AllocTests.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.alloc;
+
+import one.profiler.test.Output;
+import one.profiler.test.Assert;
+import one.profiler.test.Test;
+import one.profiler.test.TestProcess;
+import one.profiler.test.Os;
+import one.profiler.test.Jvm;
+
+public class AllocTests {
+
+    //Test is broken on OpenJ9
+    @Test(mainClass = MapReader.class, jvmArgs = "-XX:+UseG1GC -Xmx1g -Xms1g", jvm = {Jvm.HOTSPOT})
+    public void alloc(TestProcess p) throws Exception {
+        Output out = p.profile("-e cpu -d 3 -o collapsed");
+        Assert.contains(out, "G1RemSet::");
+
+        out = p.profile("--alloc 1 -d 3 -o collapsed"); //OpenJ9 breaks for alloc
+        Assert.contains(out, "java/io/BufferedReader.readLine;");
+        Assert.contains(out, "java/lang/String.split;");
+        Assert.contains(out, "java/lang/String.trim;");
+        Assert.contains(out, "java\\.lang\\.String\\[]");
+    }
+
+    @Test(mainClass = MapReaderOpt.class, enabled = false, jvmArgs = "-XX:+UseParallelGC -Xmx1g -Xms1g", jvm = {Jvm.ZING, Jvm.HOTSPOT})
```

Comment:
Can be enabled back, since you've already commented the flamegraph part out.

Reply:
Addressed